### DISCUSSION TITLE
"Request initialize failed with message: Bad file descriptor"

### DISCUSSION THREAD

**Discussion by t-rosa** [LazyVim/LazyVim#254] [Q&A] [answered]
Hello!

I got this error when I open a ts or tsx file for the first time after launching nvim :

Error executing vim.schedule lua callback: /usr/share/nvim/runtime/lua/vim/lsp.lua:1278: RPC[Error] code_name = InternalError, message = "Request initialize failed with message: Bad file descriptor"
stack traceback:
	[C]: in function 'assert'
	/usr/share/nvim/runtime/lua/vim/lsp.lua:1278: in function ''
	vim/_editor.lua: in function <vim/_editor.lua:0>

I have no idea where it comes from, do you have any idea?

**Comment by folke** [comment]
No idea. Have you checked Mason? Maybe the lsp server is not correctlt installed?

**Comment by t-rosa** [comment, marked as answer]
After reinstalling my wsl archlinux it seems to be fixed.
